src/config/feature-flags.ts

```diff
@@ -41,8 +41,8 @@ const getLaunchDarklyValue = async (flag: BooleanFlags | StringFlags, defaultVal
 		await launchdarklyClient.waitForInitialization();
 		const user = createLaunchdarklyUser(jiraHost);
 		return launchdarklyClient.variation(flag, user, defaultValue);
-	} catch (err: unknown) {
-		logger.error(`Error resolving value for feature flag "${flag}"`, err);
+	} catch (err) {
+		logger.error({flag, ...err}, "Error resolving value for feature flag");
```

Comment:
We wouldn't get stack trace if we log like that. Should be 

`logger.error(err, "Error. .... ." )`

Reply:
pretty sure we do as the stack trace is a property of the error.

Reply:
No, you can test it. We had this problem a week ago

Reply:
well, it's a good thing that the stack trace isn't helping at all in the logs :/